src/main/java/bio/terra/service/dataset/flight/upgrade/disableSecureMonitoring/DisableSecureMonitoringDeleteSourceDatasetAndSnapshotsTpsPolicyStep.java

```diff
@@ -0,0 +1,54 @@
+package bio.terra.service.dataset.flight.upgrade.disableSecureMonitoring;
+
+import bio.terra.common.iam.AuthenticatedUserRequest;
+import bio.terra.service.dataset.flight.DatasetWorkingMapKeys;
+import bio.terra.service.job.DefaultUndoStep;
+import bio.terra.service.policy.PolicyService;
+import bio.terra.service.policy.exception.PolicyServiceApiException;
+import bio.terra.service.snapshot.SnapshotService;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.FlightMap;
+import bio.terra.stairway.StepResult;
+import java.util.List;
+import java.util.UUID;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DisableSecureMonitoringDeleteSourceDatasetAndSnapshotsTpsPolicyStep
```

Comment:
Are we actually deleting anything on the source dataset? It doesn't look like we are -- I think this might just have been a bad naming of a step on my part. If I'm understanding this correctly, could we update this name for both the enable and disable tps steps?

Reply:
That's a good point since we just have policies on snapshots right now